src/css/hero.css

```diff
@@ -1,159 +1,504 @@
+source {
+  display: block;
+}
+body {
+  box-sizing: border-box;
+  font-family: 'DM Sans', sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  color: #fff;
+  background-color: #ae1332;
+}
+.h-container {
+  margin: 0px auto;
+}
+
+@media screen and (max-width: 374px) {
+  .h-container {
+    width: 100%;
+  }
+}
+
+@media screen and (min-width: 375px) {
+  .h-container {
+    max-width: 375px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .h-container {
+    max-width: 768px;
+  }
+}
+
 @media screen and (min-width: 1280px) {
+  .h-container {
+    max-width: 1280px;
+  }
+}
+
+.hero {
+  position: relative;
```

Comment:
dodaj do .hero:

padding-top:106px

Reply:
padding-top to 56px i jest dodany w .header-primary. Jak dodam go do .hero to rozjedzie się cała strona bo .hero mam relatyvne dla innych elementów z pozycją absolut.